### PR TITLE
Create docker container file and utility scripts.

### PR DESCRIPTION
**Pull request by zivadinac**

### CODE REVIEW

**Comment by zivadinac** [comment]
Take a look @igridchyn please. :)

File: lfp_online/BinFileReaderProcessor.cpp [hunk 1]
```diff
@@ -164,7 +164,8 @@ std::string BinFileReaderProcessor::binFileDuration(std::string file_path) {
 		return binFileDurationFromNSampes(fsize / 2 / buffer->CHANNEL_NUM);
 	}
 
-	return 0;
+    // return 0; --  bug?: int -> null pointer -> string (undefined behaviour)
```
Comment (zivadinac) [inline lfp_online/BinFileReaderProcessor.cpp:167]: @igridchyn The intention was to return string "0"? Am I right?

Comment (igridchyn) [reply lfp_online/BinFileReaderProcessor.cpp:167]: It should be, yes, since a string should be returned.

**Review by igridchyn — APPROVED** [review]
Awesome, thanks a lot @zivadinac !
This should make it way much easier for new users to install and deploy our software.

**Pull request merged**